src/Microsoft.AspNetCore.Razor.Tools/Internal/ResolveTagHelpersRunCommand.cs

```diff
@@ -3,9 +3,9 @@
 
 using System.Collections.Generic;
 using System.Globalization;
-using dotnet_razor_tooling;
 using Microsoft.AspNetCore.Razor;
 using Microsoft.AspNetCore.Razor.Compilation.TagHelpers;
+using Microsoft.AspNetCore.Razor.Tools;
 
 namespace Microsoft.AspNetCore.Tooling.Razor.Internal
```

Comment:
should this namespace be changed to `Microsoft.AspNetCore.Razor.Tools.Internal` ?